Sources/Confidence/Confidence.swift

```diff
@@ -1,23 +1,40 @@
 import Foundation
 
 public class Confidence: ConfidenceEventSender {
-    public var context: ConfidenceStruct
+    private let parent: ConfidenceEventSender?
+    private var localContext: ConfidenceStruct
+
+    public var context: ConfidenceStruct {
```

Comment:
if the confidence starts implementing this `ContextProvider` this can be getContext and the localContext can be renamed to `Context`.